test/suites/smoke-test-dancelight/test-inbound-channel-validation-reception.ts

```diff
@@ -42,7 +42,7 @@ describeSuite({
                     await api.query.externalValidatorSlashes.bondedEras()
                 ).toJSON() as BondedEraParams[];
                 // Let's check 2 recent eras
-                for (const bondedEra of bondedEras.slice(-2)) {
+                for (const bondedEra of bondedEras.slice(-5)) {
```

Comment:
let's maintain 2 maybe to make it a bit more lightweight

Reply:
Fixed